Hoopes process for high purity aluminium

Fiery36 opened the pull request:
Will need to increase the number of fluid inputs on the electrolytic cell

Let's also ignore the fact that the molten fluorine compounds would attack the steel vessel

Fiery36 committed:
hoopes process

WideMann commented on groovy/material/FirstDegreeMaterialsB.groovy:
```diff
@@ -662,5 +662,11 @@ class FirstDegreeMaterialsB {
+
+        BariumFluoride = new Material.Builder(8762, SuSyUtility.susyId('barium_fluoride'))
+                .dust()
+                .components(Barium, Fluroine)
```
Minor spelling mistake, also there should be 2 fluorine atoms

WideMann commented on groovy/postInit/chemistry/inorganic_chemistry/elements/p_block/group13/AluminiumChain.groovy:
```diff
@@ -223,14 +223,30 @@ BR.recipeBuilder()
+    .notConsumable(ore('dustBariumFluoride'))
+    .notConsumable(metaitem('graphite_electrode'))
+    // .notConsumable(some sort of carbon block/plate)
+    .inputs(ore('dustAluminium') * 5)
```
The patent states a ratio of aluminium to copper as 6:4, with up to 10% silicon. The copper shoudln't be consumed, and the process would need some molten aluminium too. You can also make the fluoride mixture a liquid. (The AlCuSi alloy should be too)

WideMann replied:
Also a graphite crucible would be reasonable as the second electrode

Fiery36 replied:
> The patent states a ratio of aluminium to copper as 6:4, with up to 10% silicon. The copper shoudln't be consumed, and the process would need some molten aluminium too. You can also make the fluoride mixture a liquid. (The AlCuSi alloy should be too)

I'm going off of Ullman's which makes no mention of silicon and yeah the copper should indeed be modeled as not being consumed here. 

Why would the process need to start with molten aluminium, or rather, why should we make the input molten as opposed to dust?

WideMann replied:
Molten alloy as input becasue iirc, that's what induction smelters output. And also it's closer to real life, as both Hall-Heroult and Hoopes use molten inputs. And the molten non-alloy aluminium to "catch" aluminium electrolyzed from the AlCu alloy.

Fiery36 replied:
> Also a graphite crucible would be reasonable as the second electrode

Everything I've read on this has referred to the anode as a carbon block while explicitly calling the cathode a graphite electrode. I don't know if there is any reasoning for this beyond graphite being more expensive and overkill for the purpose here.

Fiery36 replied:
> Molten alloy as input becasue iirc, that's what induction smelters output. And also it's closer to real life, as both Hall-Heroult and Hoopes use molten inputs. And the molten non-alloy aluminium to "catch" aluminium electrolyzed from the AlCu alloy.

Having not looked into the IRL Hall-Heroult I couldn't comment on that, but in susy it only has solid inputs. Also as I understand it the pure molten aluminium would simply float up and through the electrolyte layer since without copper it's less dense, so I'm not sure it's necessary to start with molten impure aluminium?

Rick8484 replied:
Wiki clearly states metals are molten and procces does not consume anything

Fiery36 replied:
The cell is operating at 900-1000 degC which is enough to melt everything

Rick8484 replied:
Remember that this is a mixture of salts not single
So they lower each other melting point
dont know about Al-Cu-Si Alloy

Fiery36 replied:
aluminium and copper are eutectic yes

WideMann reviewed (commented):
Spelling mistake to correct, and i think the recipe needs a change

Fiery36 committed:
spelling is hard

Fiery36 committed:
adjustments

Fiery36 committed:
enable liquid high purity aluminium

planetme commented on groovy/postInit/chemistry/inorganic_chemistry/elements/p_block/group13/AluminiumChain.groovy:
```diff
@@ -223,14 +223,30 @@ BR.recipeBuilder()
 
+ELECTROLYTIC_CELL.recipeBuilder()
+    .notConsumable(metaitem('graphite_electrode'))
+    .notConsumable(fluid('sodium_fluoride') * 576)
```
nvm maybe we should mix the solid first together as a new material and then melt afterwards.

planetme replied:
bump on this

bruberu commented on groovy/postInit/chemistry/inorganic_chemistry/elements/p_block/group13/AluminiumChain.groovy:
```diff
@@ -223,14 +223,30 @@ BR.recipeBuilder()
     .buildAndRegister()
 
-// Ullman's - Aluminium Ch. 5. Hall-Heroult makes like 99.9% pure already so not making it a chanced output.
-ZONE_REFINER.recipeBuilder()
```
This could be kept, if perhaps moved up a few tiers.

planetme committed:
Merge branch 'master-ceu' into pr/1735

planetme committed:
fix: some electrolysis additions

- Added membranes to water and chloralkali electrolysis method
- Remixed hoopes process electrolytes to stay under fluid input limit
- Added graphite block anode

planetme committed:
change hall heroult process to output liquid
